lib/local-links-manager/check_links/link_status_updater.rb

```diff
@@ -8,15 +8,29 @@ def initialize(link_checker = LinkChecker.new)
       end
 
       def update
+        check_all_links
+        update_local_authorities_with_broken_link_count
```

Comment:
Just thinking about this... currently the broken link counts for LAs only get updated after all links have been checked. Currently this is run at 2am every night and takes about 6.5 hours to finish (estimate based on highly unscientific guesswork), so the broken link counts would only be updated around 8:30 am. Are we happy with this sort of delay in the results? Potentially anyone using the service before 8:30 am would see wrong results...